lib/rubocop/cop/style/struct_unnecessary_keyword_init.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks for unnecessary keyword initialization in Struct.new.
+      #
+      # @example
+      #   # bad
+      #   foo = Struct.new(:bar, keyword_init: true)
+      #
+      #   # good
+      #   foo = Struct.new(:bar)
+      #
+      class StructUnnecessaryKeywordInit < Base
```

Comment:
Usually we use the word "redundant" in cop names. It'd be good for you to stay consistent with existing naming and perhaps call this `RedundantStructKeywordInit` or something along those lines.